schemas/outcomes-metrics.v1.schema.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
To what extent is the `outcomes-metrics` schema actually different from `outcomes` or is it mostly the same?

In Snuba, all of `outcomes`, `outcomes-billing` and `loadbalancer-outcomes` all share the exact same consumer code, and we just re-use the same schema in those scenarios. In the past we generally focused on having schemas reflect the minimum set of requirements for a consumer to work correctly. If the consumer is the same one, it might be simpler to use the existing outcomes schema for the new topic rather than maintain a separate one. On the other hand if the consumer code is different, I think a different schema here makes sense.

Reply:
The biggest differences to the existing outcomes are:

- Metric outcomes are tracked by MRI (consumer is expected to parse type and namespace from the MRI and materialize it in the database)
- Metric outcomes have quantity *and* cardinality tracked, where latter is aggregated max by hour instead of sum

We considered trying to make this work with the current outcomes but came to the conclusion that the metric outcomes are too different and we'd rather have a separate concept just for metrics.